docker/custom-job-images/Makefile

```diff
@@ -0,0 +1,25 @@
+VERSION ?= 0.1
+
+local: python-local duckdb-local
+
+build: python duckdb
+
+python: 
+	@$(MAKE) -C python build 
+
+python-local: 
+	@$(MAKE) -C python local
+
+duckdb: 
+	@$(MAKE) -C duckdb build 
+
+duckdb-local: 
+	@$(MAKE) -C duckdb local	
+
+.PHONY: local python duckdb
+
+
+python-test:
```

Comment:
Should we br running this as part of CI?

Reply:
We could, but we'll also need to look at how we push the images.  At the moment it requires a manual push to docker hub.